runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -161,12 +161,16 @@ export function get_page_handler(
 
 		try {
 			const root_preload = manifest.root_comp.preload || (() => {});
-			const root_preloaded: PreloadResult = root_preload.call(preload_context, {
+			const root_preloaded: PreloadResult = root_preload.call(
+				preload_context,
```

Comment:
why is anything under `runtime/src` changing? I think this file was already linted and this change isn't really necessary. Personally, I prefer the previous indentation on this file, which was more concise

Reply:
Not sure. I probably ran prettier over this block for whatever reason. I'll roll it back.